code/game/objects/items/plushes.dm

```diff
@@ -66,8 +107,143 @@
 		return
+	if(istype(I, /obj/item/toy/plush))
+		love(I, user)
+		return
 	return ..()
 
+/obj/item/toy/plush/proc/love(obj/item/toy/plush/Kisser, mob/living/user)	//~<3
+	var/chance = 100	//to steal a kiss, surely there's a 100% chance no-one would reject a plush such as I?
+	var/concern = 20	//perhaps something might cloud true love with doubt
+	var/loyalty = 30	//why should another get between us?
+	var/duty = 50		//conquering another's is what I live for
+
+	//we are not catholic
+	if (young == TRUE || Kisser.young == TRUE)
+		user.show_message("<span class='notice'>[src] plays tag with [Kisser].</span>", 1,
+			"<span class='notice'>They're happy.</span>", 0)
+	//never again
+	else if(Kisser in scorned)
+		//message, visible, alternate message, neither visible nor audible
+		user.show_message("<span class='notice'>[src] rejects the advances of [Kisser]!</span>", 1,
+			"<span class='notice'>That didn't feel like it worked.</span>", 0)
+	else if(src in Kisser.scorned)
+		user.show_message("<span class='notice'>[Kisser] realises who [src] is and turns away.</span>", 1,
+			"<span class='notice'>That didn't feel like it worked.</span>", 0)
+
+	//first comes love
+	else if(Kisser.lover != src && Kisser.partner != src)	//cannot be lovers or married
+		if(Kisser.lover)	//if the initiator has a lover
+			Kisser.lover.heartbreak(Kisser)	//the old lover can get over the kiss-and-run whilst the kisser has some fun
+			chance -= concern	//one heart already broken, what does another mean?
+		if(lover)	//if the recipient has a lover
+			chance -= loyalty	//mustn't... but those lips
+		if(partner)	//if the recipient has a partner
+			chance -= duty	//do we mate for life?
+
+		if(prob(chance))	//did we bag a date?
+			user.visible_message("<span class='notice'>[user] makes \the [Kisser] kiss \the [src]!</span>",
+									"<span class='notice'>You make \the [Kisser] kiss \the [src]!</span>")
+			if(lover)	//who cares for the past, we live in the present
+				lover.heartbreak(src)
+			new_lover(Kisser)
+			Kisser.new_lover(src)
+		else
+			user.show_message("<span class='notice'>[src] rejects the advances of [Kisser], maybe next time?</span>", 1,
+								"<span class='notice'>That didn't feel like it worked, this time.</span>", 0)
+
+	//then comes marriage
+	else if(Kisser.lover == src && Kisser.partner != src)	//need to be lovers (assumes loving is a two way street) but not married (also assumes similar)
+		user.visible_message("<span class='notice'>[user] pronounces \the [Kisser] and \the [src] married! D'aw.</span>",
+									"<span class='notice'>You pronounce \the [Kisser] and \the [src] married!</span>")
+		new_partner(Kisser)
+		Kisser.new_partner(src)
+
+	//then comes a baby in a baby's carriage, or an adoption in an adoption's orphanage
+	else if(Kisser.partner == src && !src.plush_child)	//the one advancing does not take ownership of the child and we have a one child policy in the toyshop
```

Comment:
you shouldn't need src for src.plush_child

Reply:
Shhh I totally left that in to make sure maintainers are doing their job and totally didn't get told this by MSO in coderbus before I typed it in.

Here's your gold star 🌟